pkg/protocol/engine/booker/inmemorybooker/booker.go

```diff
@@ -173,6 +192,39 @@ func (b *Booker) markInvalid(block *blocks.Block, err error) {
 	}
 }
 
+func (b *Booker) inheritConflicts(block *blocks.Block) (conflictIDs *advancedset.AdvancedSet[iotago.TransactionID], err error) {
+	conflictIDsToInherit := advancedset.New[iotago.TransactionID]()
+
+	for _, parent := range block.ModelBlock().ParentsWithType() {
+		parentBlock, exists := b.blockCache.Block(parent.ID)
+		if !exists {
+			return nil, errors.Errorf("parent %s does not exist", parent.ID)
+		}
+
+		switch parent.Type {
+		case model.StrongParentType:
+			conflictIDsToInherit.AddAll(parentBlock.ConflictIDs())
```

Comment:
Here you should inherit the parent's block PayloadConflicts as well, and you have toset these conflicts separately: otherwise you cannot inherit only the payload conflicts separetely.